macros/anaUltraScurve.py

```diff
@@ -290,7 +297,40 @@ def overlay_fit(VFAT, CH):
 canv.SaveAs(filename+'/Summary.png')
 
 if options.SaveFile:
+    confF = open(filename+'/chConfig.txt','w')
+    confF.write('vfatN/I:vfatCH/I:trimDAC/I:mask/I\n')
+    for vfat in range (0,24):
+        for ch in range (0, 128):
+            confF.write('%i\t%i\t%i\t%i\n'%(vfat,ch,trim_list[vfat][ch],masks[vfat][ch]))
+            pass
+        pass
+    confF.close()
     outF.cd()
     myT.Write()
     outF.Close()
     pass
+
+
+
+
+
+
+
```

Comment:
How much do we really care if there are some blank lines at the bottom of the file?